data/data/aws/main.tf

```diff
@@ -9,6 +9,14 @@ locals {
 
 provider "aws" {
   region = var.aws_region
+
+  endpoints {
+    s3 = lookup(var.custom_endpoints, "s3", null)
+    ec2 = lookup(var.custom_endpoints, "ec2", null)
+    elb =  lookup(var.custom_endpoints, "elb", null)
+    iam =  lookup(var.custom_endpoints, "iam", null)
+    route53 =  lookup(var.custom_endpoints, "route53", null)
```

Comment:
terraform linting issue: these equal signs and values should be lined up.

Reply:
Updated terraform linting.